app/src/main/res/values/strings.xml

```diff
@@ -95,4 +95,10 @@
     <string name="reject">Reject</string>
     <string name="select_all">Select all</string>
     <string name="see_more">See More..</string>
+    <string name="user">User :</string>
+    <string name="products">Product :</string>
```

Comment:
WARNING: [UnusedResources] The resource `R.string.products` appears to be unused
 Unused resources make applications larger and slow down builds.